roles/network_plugin/calico/files/openssl.conf

```diff
@@ -0,0 +1,19 @@
+req_extensions = v3_req
+distinguished_name = req_distinguished_name
+
+[req_distinguished_name]
+
+[ v3_req ]
+basicConstraints = CA:FALSE
+keyUsage = digitalSignature, keyEncipherment
+
+[ ssl_client ]
+extendedKeyUsage = clientAuth, serverAuth
```

Comment:
It's recommended to not have such 'generic' usage where server and client certs are interchangeable.
It'd be better to declare distinct `extendedKeyUsage` for server and client.

Reply:
I don't see that this is a big problem.